app/database/migration/server/index.ts

```diff
@@ -4,13 +4,19 @@
 // NOTE : To implement migration, please follow this document
 // https://nozbe.github.io/WatermelonDB/Advanced/Migrations.html
 
-import {addColumns, createTable, schemaMigrations} from '@nozbe/watermelondb/Schema/migrations';
+import {addColumns, createTable, schemaMigrations, unsafeExecuteSql} from '@nozbe/watermelondb/Schema/migrations';
 
 import {MM_TABLES} from '@constants/database';
 
 const {CHANNEL_BOOKMARK, CHANNEL_INFO, DRAFT, POST} = MM_TABLES.SERVER;
 
 export default schemaMigrations({migrations: [
+    {
+        toVersion: 6,
+        steps: [
+            unsafeExecuteSql('CREATE INDEX Post_type ON Post (type)'),
```

Comment:
I wonder if it would be safer to write it like this
```suggestion
            unsafeExecuteSql('CREATE UNIQUE INDEX IF NOT EXISTS Post_type ON Post (type)'),
```

We make sure the index is unique and cannot be duplicate, only gets created if it does not exist

for more info see https://www.sqlite.org/lang_createindex.html

Reply:
My SQL is a bit rusty, so I may be mixing concepts, but... the index being unique won't make it so you cannot add two posts with the same type? That is what I understand from the documentation, but again, I may be misunderstanding this.

Reply:
@enahum @larkox I don't think it should be `UNIQUE`, because type is not unique in each row,

In fact, this is what would happened if it is unique:

```
Error: UNIQUE constraint failed: Post.type
    at setUpWithMigrations (native)
```

As testing this with existing db (2 of them actually for 2 different servers), I can confirm that without unique, the creation of indexes (in both dbs) was successful.

Reply:
@rahimrahman Right. From what I've gathered, the `UNIQUE` keyword adds constraint to the inserted values, not the index itself.

Also very rusty with SQL haha. Learned something new today 👍